addons/base_vat/models/res_partner.py

```diff
@@ -29,6 +30,8 @@
     'be': 'BE0477472701',
     'bg': 'BG1234567892',
     'ch': 'CHE-123.456.788 TVA or CH TVA 123456',  # Swiss by Yannick Vaucher @ Camptocamp
+    'cl': 'CL76086428-5',
+    'co': 'CO213123432-1 or CO213.123.432-1',
```

Comment:
Both examples don't work.

```
>>> stdnum.util.get_cc_module('co','vat').is_valid("CO213.123.432-1")
False
>>> stdnum.util.get_cc_module('co','vat').is_valid("CO213123432-1")
False
```

Reply:
@nim-odoo remove CO from starting
It's work in odoo because when odoo fail in 1st test then it tests it again with remove country code from the vat.